test/memory_model_md.ll

```diff
@@ -0,0 +1,31 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv %t.bc -o %t.spv
+; RUN: llvm-spirv %t.spv -o %t.spt --to-text
+; RUN: llvm-spirv -r %t.spv -o %t.bc
+; RUN: llvm-dis %t.bc -o %t.ll
+; RUN: FileCheck %s --input-file %t.ll  -check-prefix=LLVM
+; RUN: FileCheck %s --input-file %t.spt -check-prefix=SPV
+
+; ModuleID = 'float_control_empty.bc'
+source_filename = "float_control_empty.cpp"
+target datalayout = "e-p:64:64-i64:64-n8:16:32"
+target triple = "spir"
+
+; LLVM-DAG: !spirv.MemoryModel = !{![[MMMD:[0-9]+]]}
```

Comment:
The use of `-DAG` feels a bit too strong for this case.

Reply:
Fixed